js/src/stats.js

```diff
@@ -95,6 +95,26 @@ module.exports = (common) => {
       })
     })
 
+    it('.bw Poll', (done) => {
+      if (!withGo) {
+        console.log('Not supported in js-ipfs yet')
+        return done()
+      }
+
+      ipfs.stats.bw({poll: true}, (err, res) => {
+        expect(err).to.not.exist()
+        expect(res).to.exist()
+
+        res.once('data', (data) => {
```

Comment:
We should have a way of closing the stream as well, for when you're done polling for stats

Reply:
@VictorBjelkholm probably `stream.destroy()` is our best shot

Reply:
Added `res.destroy()` in the end.